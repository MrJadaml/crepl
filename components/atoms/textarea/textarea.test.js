import React from 'react';
import { Textarea } from './index';
import { cleanup, render } from '@testing-library/react';
import { getByText } from '@testing-library/dom';

afterEach(cleanup);

describe('Textarea', () => {
  it('should render a textarea element', () => {
    const { asFragment } = render(
      <Textarea value="Bob has a lot to say"></Textarea>
    );

    expect(asFragment()).toMatchSnapshot();
  });

  it('should render children', () => {
    const { getByText } = render(
      <Textarea value="Carl has a lot to say"></Textarea>
    );

    getByText('Carl has a lot to say');
  });
});
