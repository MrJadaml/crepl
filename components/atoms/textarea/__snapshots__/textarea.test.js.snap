// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Textarea should render a textarea element 1`] = `
<DocumentFragment>
  <textarea
    class="textarea"
  >
    Bob has a lot to say
  </textarea>
</DocumentFragment>
`;
